src/server.coffee

```diff
@@ -14,6 +17,7 @@ config.alerts = config.get('alerts')
 config.polling = config.get('polling')
 config.reports = config.get('reports')
 config.auditing = config.get('auditing')
+config.agenda = config.get('agenda')
 
 mongoose = require "mongoose"
 exports.connectionDefault = connectionDefault = mongoose.createConnection(config.mongo.url)
```

Comment:
Do the mongo connections still need run if the node is master? I believe this will start up a connection pool, so it might save some resources by excluding this